vector/src/main/java/im/vector/app/core/pushers/UnifiedPushHelper.kt

```diff
@@ -92,16 +92,21 @@ class UnifiedPushHelper @Inject constructor(
                 return@launch
             }
 
-            // By default, use internal solution (fcm/background sync)
-            UnifiedPush.saveDistributor(context, context.packageName)
```

Comment:
preemptively setting the distributor without registering the app causes the user selection to be skipped if it matches https://github.com/vector-im/element-android/pull/6890/files#diff-d0af9539e5621c00bfa7f4315721ab35c43af91f06e81d8f8a8b097d51a45487L152